pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseRealtimeClusterIntegrationTest.java

```diff
@@ -192,6 +192,7 @@ public void tearDown()
       throws Exception {
     dropRealtimeTable(getTableName());
     waitForTableDataManagerRemoved(TableNameBuilder.REALTIME.tableNameWithType(getTableName()));
```

Comment:
is this still required?

Reply:
Kept it here to ensure table data managers are removed